src/lib/pipeline.ml

```diff
@@ -140,7 +154,8 @@ module Construct = struct
 
   let hisat fastq = Hisat fastq
 
-  let stringtie bam = Stringtie bam
+  let stringtie ?(use_reference_gtf =`If_available) bam =
```

Comment:
It's ``[`Yes, `No, `If_available]``: https://github.com/hammerlab/biokepi/pull/123/files#diff-410ac8cc1ab382d8fa10f46e94f1fbb6R5